src/olympia/migrations/938-drop-l10n-from-bio-user-profile.sql

```diff
@@ -0,0 +1,23 @@
+/* FOREIGN KEY is prod's - you will need to change this for your local DB */
+
+/* Add the new column and drop the constraint so we can clear the translations.*/
+ALTER TABLE `users`
+    ADD COLUMN `_bio` longtext NULL,
+    DROP FOREIGN KEY `users_ibfk_1`;
+
+/* Copy across the translated values.  There are some users that have multiple
+localizations so we're going to use the default locale (lang) localization. */
+UPDATE `users`
+    SET `_bio` = (
+        SELECT `localized_string` FROM `translations`
+        WHERE `id`=`bio` AND `locale`=`lang`);
+
+/* Clear out the now obsolete translations. */
+DELETE FROM `translations` WHERE `id` in (SELECT `bio` FROM `users`);
+
+/* Drop the old columns and rename bio_ to take the place of bio. */
+ALTER TABLE `users`
+    DROP COLUMN `bio`,
+    DROP COLUMN lang,
+    DROP COLUMN region,
+    CHANGE COLUMN `_bio` `bio` longtext NULL;
```

Comment:
Yeah, it's a risk.  We could change the model to refer to `_bio` and not do `CHANGE COLUMN _bio bio ... ;` (and not drop the old columns).  But we're moving the pain to a later point because at some point we need to do a rename and the model and db won't match during the deploy then.